pkg/detectors/postgres/postgres_test.go

```diff
@@ -24,9 +25,9 @@ var postgresDockerHash string
 
 const (
 	postgresUser = "postgres"
-	postgresPass = "23201dabb56ca236f3dc6736c0f9afad"
+	postgresPass = "23201da=b56ca236f3dc6736c0f9afad"
 	postgresHost = "localhost"
-	postgresPort = "5433"
+	postgresPort = "5434" // Do not use 5433, as local dev environments can use it for other things
 
 	inactiveUser = "inactive"
```

Comment:
I think this is unused.

Reply:
same with inactivePort. I know you didn't touch these, but figured while i'm here i'd call it out.